packages/flutter_tools/test/general.shard/build_system/filestore_test.dart

```diff
@@ -0,0 +1,141 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter_tools/src/base/file_system.dart';
+import 'package:flutter_tools/src/base/logger.dart';
+import 'package:flutter_tools/src/build_system/build_system.dart';
+import 'package:flutter_tools/src/build_system/filestore.dart';
+import 'package:flutter_tools/src/globals.dart';
+import 'package:mockito/mockito.dart';
+
+import '../../src/common.dart';
+import '../../src/testbed.dart';
+
+void main() {
+  Testbed testbed;
+  Environment environment;
+
+  for (bool timestampMode in <bool>[true, false]) {
+    setUp(() {
```

Comment:
The `setUp()` function doesn't appear to depend on the loop variable. Can you hoist it out?

Reply:
Done